ci/docs/build.sh

```diff
@@ -26,9 +26,10 @@ nvidia-smi
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
+
 # TODO: Move installs to docs-build-env meta package
-gpuci_conda_retry install -c anaconda markdown beautifulsoup4 jq
-pip install sphinx-markdown-tables
+gpuci_conda_retry install -c conda-forge markdown beautifulsoup4 jq
+python -m pip install sphinx-markdown-tables nbsphinx recommonmark numpydoc
```

Comment:
Would it make sense to put all of these in the `conda install` command above given they are all in conda-forge?